doc/Developing/SNMP-Traps.md

```diff
@@ -3,13 +3,86 @@ path: blob/master/doc/
 
 # Creating snmp trap handlers
 
-Create a new class in LibreNMS\Snmptrap\Handlers that implements the
-LibreNMS\Interfaces\SnmptrapHandler interface. 
+You must to have working snmptrapd. See [SNMP TRAP HANDLER](../Extensions/SNMP-Trap-Handler.md) 
 
-Register the mapping in the config/snmptraps.php file. Make sure to use the full trap oid.
+If you are adding trap handler for new OS, where are you using OS MIBs then you need to add MIB dir to your snmptrapd service, if it's using only MIBs from directory `/opt/librenms/mibs`.
+
+```
+ExecStart=/usr/sbin/snmptrapd -f -M /opt/librenms/mibs -m ALL
```

Comment:
This conflicts with how our docs describe to set up snmptrapd.  Perhaps we should just link to that with some extra notes: https://docs.librenms.org/Extensions/SNMP-Trap-Handler/#configure-snmptrapd